src/qml/AboutPage.qml

```diff
@@ -29,6 +30,19 @@ Flickable {
         id: diskUsage
     }
 
+    Process {
+        id: process
+        onReadyRead: uptime.text = readAll();
```

Comment:
onFinished sounds like a better signal that onReadyRead which would trigger any time a command outputs anything to stdout (and here your readAll() suggests you're interested in the full output only)